packages/circuit-ui/components/TopNavigation/components/ProfileMenu/ProfileMenu.spec.tsx

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright 2021, SumUp Ltd.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { axe, render, renderToHtml } from '../../../../util/test-utils';
+import { PopoverProps } from '../../../Popover';
+
+import { ProfileMenu } from './ProfileMenu';
+
+describe('ProfileMenu', () => {
+  const baseProps = {
+    userName: 'Jane Doe',
```

Comment:
Nitpick, I would have gone for `username` because it's normally used as a single word, but that't fine as well

Reply:
I'm on the fence with this one. I wanted to align it with the other `user*` props 🤷🏻‍♂️ Happy to change it.

Reply:
That makes sense too 👍